sql/src/test/java/io/crate/integrationtests/ArithmeticIntegrationTest.java

```diff
@@ -394,4 +395,73 @@ public void testSelectOrderByScalarOnNullValue() throws Exception {
         execute("select (d - 10) from t order by (d - 10) nulls first limit 2");
         assertThat(response.rows()[0][0], is(nullValue()));
     }
+
+    @Test
+    public void test_add_interval_to_interval() {
```

Comment:
I think we should move these to a unit test using `AbstractScalarFunctionsTest`, it'll be at least an order of magnitude faster to run them then.